binding/angular/projects/rhino-web-angular/src/lib/rhino.service.ts

```diff
@@ -85,17 +54,7 @@ export class RhinoService implements OnDestroy {
     }
   }
 
-  public async release(): Promise<void> {
-    if (this.rhinoWorker !== null) {
-      this.rhinoWorker.postMessage({ command: 'release' });
-    }
-    if (this.webVoiceProcessor !== null) {
-      await this.webVoiceProcessor.release();
-    }
-    this.isInit = false;
-  }
-
-  public pushToTalk(): boolean {
+  public async pushToTalk(): Promise<boolean> {
```

Comment:
Even though `pushToTalk` is a valid name here - we simply call this `process` in the Manager classes for mobile. I slightly prefer `process` because it's generic and doesn't prescribe a particular behaviour

Reply:
Addressed as discussed